lib/chai/interface/assert.js

```diff
@@ -732,6 +732,26 @@ module.exports = function (chai, util) {
     new Assertion(val, msg).to.not.be.a('number');
   };
 
+   /**
+   * ### .isFinite(value, [message])
+   *
+   * Asserts that `value` is a finite number. This will fail for NaN, unlike `.isNumber`
+   * Asserts that `value` is a finite number. Unlike `.isNumber`, this will fail for `NaN` and `Infinity`
+   *
+   *     var cups = 2;
+   *     assert.isFinite(cups, 'how many cups');
```

Comment:
Since the main goal of this assertion is to prove something is not `NaN` or `Infinity`, it would be nice to have an example using `.not` and these two values, just to make the docs more clear.